schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/dao/common/OrganizationDao.java

```diff
@@ -73,4 +78,20 @@ public void delete(@Nonnull Key id) {
             }
         });
     }
+
+    public List<Organization> forUser(User user) throws EntityNotFoundException {
+        return forUser(user.getId());
+    }
+
+    public List<Organization> forUser(Key userId) throws EntityNotFoundException {
+        OrganizationMemberDao organizationMemberDao = new OrganizationMemberDao();
+        List<OrganizationMember> organizationMembers = organizationMemberDao.byUserId(userId);
+        if (organizationMembers.isEmpty()) return Collections.emptyList();
+        List<Key> organizationIds = new ArrayList<>();
+        for (OrganizationMember organizationMember : organizationMembers) {
+            Key key = organizationMember.getOrganizationRef().getKey();
+            if (key != null) organizationIds.add(key);
+        }
+        return get(organizationIds);
+    }
 }
```

Comment:
This is not the MOST optimal.  We will end up calling get on memcache 3 times if the data is still on cache
 1. OrganizationMemberDao -> Memcache to get the cached query result for (userId): returns List<Key> with the ids of the OrganizationMember(s)
 2. OrganizationMemberDao -> Memcache getall with result from 1.
 3. OrganizationDao -> Memcache getall with the keys of the objects returned from 2.

I still think its way more efficient than quering the datastore